README.md

```diff
@@ -1,11 +1,18 @@
 <!-- markdownlint-disable-next-line MD041 -->
```

Comment:
this ignore is here because the first line is supposed to be an `H1`. If there's a line of badges first, I think that's ok, but putting several lines of text + badges would be an anti-pattern and would fail `markdownlint` multiple times. By convention, any text should be inside of / below headings